### DISCUSSION TITLE
Controlling a bot then switching back to main character deletes bot when either options "Save and quit" or "Server Lobby" is used.

### DISCUSSION THREAD

**Discussion by ieatmen4088** [FakeFishGames/Barotrauma#15199] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I used the setclientcharacter command to switch to a bot to craft something, then switched back to my main character and then pressed save and quit. After restarting the server, the bot I used to craft something disappeared and was pretty much deleted (I used the save decompressor to check). I went to test it out again and it happened again, only happens if you close the server without going to the next location.

edit: read line "My issue happened while using mods wrongly"

### Reproduction steps

1. setclientcharacter to some random bot
2. setclientcharacter back to your original self
3. Return to the server lobby or save and quit the save **without** embarking on a destination.
4. Reload the save.
5. Bot that was controlled is gone.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by TorlanDelta** [comment]
Same thing is happening to me, it's very frustrating. Whenever I need bots to do some crafting, I have to backup beforehand and manually reinsert the chars via xml and save decompressor.

**Comment by greenking13** [comment]
I can confirm the character deletion happens in the current version. v1.11.4.1

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14333
